hack/whitespace.sh

```diff
@@ -9,14 +9,41 @@ function fix_one_file() {
 	rm -f ${fname}.tmp
 }
 
+function check_files {
+	while read fname; do
+		IDEAL_FILE=$(sed --follow-symlinks 's/[[:space:]]*$//' ${fname} | sed 'N;/^\n$/D;P;D;')
+		REAL_FILE=$(cat ${fname})
+
+		if [[ "$IDEAL_FILE" != "$REAL_FILE" ]]; then
+			echo "need whitespace fix for file $fname"
+			#diff -u $(echo "$REAL_FILE") $(echo "$IDEAL_FILE")
```

Comment:
remove commented line please

Reply:
i put in the diff - now make whitespace-check is displaying the problem cass.